From 514385d5966ce5e43474122645be4c8461028e0f Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 6 May 2019 15:26:41 +0700
Subject: [PATCH 089/248] arm64: dts: renesas: Add HiHope RZ/G2M extension
 board support

The HiHope RZ/G2M extension board is made of HiHope RZ/G2M platform
(the main board) and the sub board that goes on top of main board.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/Makefile                     |  3 ++-
 arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts | 16 ++++++++++++++++
 2 files changed, 18 insertions(+), 1 deletion(-)
 create mode 100644 arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts

diff --git a/arch/arm64/boot/dts/renesas/Makefile b/arch/arm64/boot/dts/renesas/Makefile
index cd986fa..0c85a14 100644
--- a/arch/arm64/boot/dts/renesas/Makefile
+++ b/arch/arm64/boot/dts/renesas/Makefile
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0
-dtb-$(CONFIG_ARCH_R8A774A1) += r8a774a1-hihope-rzg2m.dtb
+dtb-$(CONFIG_ARCH_R8A774A1) +=	r8a774a1-hihope-rzg2m.dtb \
+				r8a774a1-hihope-rzg2m-ex.dtb
 dtb-$(CONFIG_ARCH_R8A774C0) += r8a774c0-cat874.dtb r8a774c0-ek874.dtb
 dtb-$(CONFIG_ARCH_R8A7795) += r8a7795-salvator-x.dtb r8a7795-h3ulcb.dtb
 dtb-$(CONFIG_ARCH_R8A7795) += r8a7795-h3ulcb-kf.dtb
diff --git a/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts
new file mode 100644
index 0000000..8bee8b2
--- /dev/null
+++ b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the Hoperun Technology HiHope RZ/G2M extension board
+ * (hihope-rzg2m-ex)
+ *
+ * Copyright (C) 2019 Renesas Electronics Corp.
+ */
+
+#include "r8a774a1-hihope-rzg2m.dts"
+
+/ {
+	model = "Hoperun Technology HiHope RZ/G2M extension board \
+		(hihope-rzg2m-ex)";
+	compatible =	"hoperun,hihope-rzg2-ex", "hoperun,hihope-rzg2m",
+			"renesas,r8a774a1";
+};
-- 
2.7.4

